RestrictEvents/SoftwareUpdate.cpp

```diff
@@ -159,6 +159,13 @@ static int my_sysctl_vmm_present(__unused struct sysctl_oid *oidp, __unused void
 	return FunctionCast(my_sysctl_vmm_present, org_sysctl_vmm_present)(oidp, arg1, arg2, req);
 }
 
+static mach_vm_address_t org_sysctl_f16c; // hw.optional.f16c
```

Comment:
Why do you save org function when you never call it?

Reply:
Good catch, was originally performing a process check before realizing library names wouldn't be used
Will fix shortly